students/GKim/lesson04/dict_lab.py

```diff
@@ -0,0 +1,110 @@
+"""
+Dictionaries 1
+Create a dictionary containing “name”, “city”, and “cake” for “Chris” from “Seattle” who likes 
+“Chocolate” (so the keys should be: “name”, etc, and values: “Chris”, etc.)
+Display the dictionary.
+Delete the entry for “cake”.
+Display the dictionary.
+Add an entry for “fruit” with “Mango” and display the dictionary.
+Display the dictionary keys.
+Display the dictionary values.
+Display whether or not “cake” is a key in the dictionary (i.e. False) (now).
+Display whether or not “Mango” is a value in the dictionary (i.e. True).
+
+Dictionaries 2
+Using the dictionary from item 1: Make a dictionary using the same keys but with the number of ‘t’s 
+in each value as the value (consider upper and lower case?).
+
+Sets
+Create sets s2, s3 and s4 that contain numbers from zero through twenty, divisible by 2, 3 and 4.
+Display the sets.
+Display if s3 is a subset of s2 (False)
+and if s4 is a subset of s2 (True).
+
+Sets 2
+Create a set with the letters in ‘Python’ and add ‘i’ to the set.
+Create a frozenset with the letters in ‘marathon’.
+display the union and intersection of the two sets.
+
+"""
+
+
+
+
+student_info = {"name": "George", "city": "Kent", "cake": "Chocolate"}
+student_info2 = {"name": "George", "city": "Kent", "cake": "Chocolate"}
+
+
+def show_dict(dct):
+    for k, v in dct.items():
+        print("{:<1}: {:<10}".format(k, v))
+    print("\n")
+    return 
+
+def dictionaries_1(dct):
+
+    print("\nDictionary")
+    show_dict(dct)
+    dct.pop("cake")
+    print("\nDictionary with cake removed")
+    show_dict(dct)
+    dct.update(fruit = "Mango")
+    print("\nDictionary with 'fruit' added")
+    show_dict(dct)
+    print("\n{x} {y} {x}".format( x = "*"*10, y = "END"))
+       
+    if "cake" in dct:
+        print("GOT ME SOME!")
+    else:
+        print("NO CAKE!!")
+    
+    
+    if "Mango" in dct.values():
+        print("I got some Mango's!!!\n")
+    else:
+        print("no mango's here")
+
+    
+def dictionaries_2(dct):
+    
+    for key in dct.keys():
```

Comment:
You actually don't need to specify the .keys(), you can just do
for key in dct: